_posts/2016-11-29-future.md

```diff
@@ -0,0 +1,199 @@
+
+Users should be able to easily use features before they are actually released in browsers in stage 4.
+
+This is the core of Babel. It had it's start as a JavaScript transpiler for ES6 back when it was still called 6to5.
+
+In short, proposal champions won't be getting much feedback if no one is actually trying them out in actual code. Babel is a fundamentally important step to the TC-39 process given familiarity in the ecosystem, a simple config change in most people's build process rather than through browser flags, and relative ease of implementation vs. native implementation.
+
+This isn't just a one-time thing as we've found out, especially for early stage proposals. The creation of a plugin means the eventual maintenance of that same plugin. Proposals could completely change in syntax or even get dropped. Since TC-39 meets every 2 months, the plugins should be updated as meetings happen so that users can be in sync and involved with the standards process.
+
+> Some transforms are simply out of date (one example is decorators) and Logan had to make [babel-plugin-transform-decorators-legacy](https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy) for the previous version and no one has had time for an update.
+
+> A good example: [babel/babel#3473](https://github.com/babel/babel/pull/3473) wasn't merged for so long because just didn't have time to review it. By the time it was merged it had already moved from stage-2 to stage-3.
+
+### Maintaining other ecosystem plugins: like JSX/Flow.
+
+There is both a React/Flow team at Facebook and they have many issues/PRs associated with Babel that we have to review/maintain.
+
+- [React plugins](https://babeljs.io/docs/plugins/#react)
+- [Flow plugin](https://babeljs.io/docs/plugins/transform-flow-strip-types)
+
+### Autoprefixer for Babel via [babel-preset-env](https://github.com/babel/babel-preset-env)
+
+This is something that is very important as we move forward to a point where users are compiling a specific set of transforms while the browsers support a specific set of features. There is a constant sliding window. This preset picks the right amount of plugins supported by the target environments so that the code can be fast/smaller.
+
+The next big feature would be dynamic checking of polyfills as well as syntax as described in [babel/babel-preset-env#20](https://github.com/babel/babel-preset-env/issues/20).
+
+### Linting via [babel-eslint](https://github.com/babel/babel-eslint)
+
+As long as you want to lint 
+with experimental features (stage-x, types) that ESLint proper won't support yet.
+
+Future: Better interop with ESLint (hook into scope, traversal), possibility of writing an ESLint rule with Babel APIs, running ESLint rules as plugins/presets?
+
+### Minification
+
+[babel/babili](https://github.com/babel/babili) was created as a future proof minifier.
```

Comment:
Why was this never changed to [`babel-minify`](https://github.com/babel/babili/issues/124)? 😦